dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -136,7 +138,7 @@ const LiveGrid = ({ children }: { children: React.ReactNode }) => (
 				${from.wide} {
 					grid-template-columns: 219px 1px 1fr 340px;
 					grid-template-areas:
-						'lines 		border media right-column'
+						'lines 		border matchtabs right-column'
```

Comment:
It's hard to be sure, but I think the spacing is off below this line. Do we need more spacing before `right-column`?